language/move-vm/runtime/src/runtime.rs

```diff
@@ -156,11 +159,14 @@ impl<'alloc> VMRuntime<'alloc> {
     ) -> VMResult<StructType> {
         let module = self.code_cache.get_loaded_module(module_id, context)?;
         let struct_idx = module.get_struct_def_index(name)?;
+        let def = module.struct_def_at(*struct_idx);
+        let handle = module.struct_handle_at(def.struct_handle);
+        verify_ty_args(&handle.type_formals, ty_args)?;
```

Comment:
@dariorussi This is the place I was mentioning to you. Please let me know if it's right to do the kind verification here.